Docs/Data Sharing Framework/Creating Custom Components.md

```diff
@@ -0,0 +1,55 @@
+Creating Custom Components
+==========================
```

Comment:
Not an issue, just genuine curiosity: is there a reason you prefer using `=========` and `--------` rather than `#` and `##`?

Reply:
I don't prefer it, but https://domchristie.github.io/turndown/ sure does, so I can swap those out. That site is what I used to convert the HTML, and it put those in there.

Reply:
![image](https://user-images.githubusercontent.com/31290917/89309541-24606b00-d639-11ea-90dd-34f6c9c196d8.png)
Ah, it was an option sadly.

Reply:
Didn't realize you had automatically converted these. Thanks for the link to that site!